tests/integration/ceph_base_object_test.go

```diff
@@ -199,6 +201,13 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 				time.Sleep(5 * time.Second)
 				continue
 			}
+			logger.Info("objectstore status is", objectStore.Status)
+			if objectStore.Status.BucketStatus.Health == cephv1.ConditionFailure && CephObjectStoreCIRetryCount == 0 {
+				CephObjectStoreCIRetryCount = CephObjectStoreCIRetryCount + 1
+				createCephObjectStore(s, helper, k8sh, namespace, storeName, 3, false)
+				testObjectStoreOperations(s, helper, k8sh, namespace, storeName)
+				break
```

Comment:
The continue would allow to retry, right?
```suggestion
				continue
```